lms/templates/instructor/instructor_dashboard_2/course_info.html

```diff
@@ -73,6 +73,22 @@
       <b>${_("No")}</b>
       %endif
     </li>
+
+    <li class="field text is-not-editable" id="field-course-start-date">
+      <label for="course-start-date">${_("Course Start Date:")}</label>
+      <b>${ section_data['start_date'] }</b>
+    </li>
+
+    <li class="field text is-not-editable" id="field-course-end-date">
+      <label for="course-end-date">${_("Course End Date:")}</label>
+      <b>${ section_data['end_date'] }</b>
```

Comment:
I think it would make sense to put these two (course start & end dates) above the "has the course started/ended" questions, to keep all the factual course data together.